hipay.php

```diff
@@ -57,7 +58,7 @@ public function __construct()
 	{
 		$this->name = 'hipay';
 		$this->tab = 'payments_gateways';
-		$this->version = '1.6.9';
+		$this->version = HIPAY_VERSION;
```

Comment:
With this change, some features could not work properly. The [validator](http://validator.prestashop.com) warns about this point btw.

Bring back the value on this line, on then call it when needed with `$this->version`